cfgmgr/buffermgr.cpp

```diff
@@ -34,6 +34,7 @@ void BufferMgr::readPgProfileLookupFile(string file)
     ifstream infile(file);
     if (!infile.is_open())
     {
+        m_pgfile_exist = false;
         return;
```

Comment:
Should we log an error here?
SWSS_LOG_ERROR

Reply:
It is not necessarily an error if we don't need this feature.  I can put an NOTICE or WARNING log message here.

Reply:
If you are sure the feature is optional, then a WARNING should be ok.

---
In reply to: [189676820](https://github.com/Azure/sonic-swss/pull/509#discussion_r189676820) [](ancestors = 189676820)

Reply:
Done.